browser/js/app.js

```diff
@@ -8,6 +8,19 @@ app.config(function ($urlRouterProvider, $locationProvider) {
     $urlRouterProvider.otherwise('/');
 });
 
+// Create cookie controller
+
+app.controller('CookieCtrl', ['$scope','$cookieStore', function($scope, $cookieStore) {
```

Comment:
is $cookieStore deprecated? https://docs.angularjs.org/api/ngCookies/service/$cookieStore